react-client/src/Components/App.js

```diff
@@ -86,12 +86,15 @@ function App() {
         "https://www.thefreedictionary.com/"
       );
     } else {
+      if (window.location.href.includes("localhost")) {
+        document.getElementById("samplePage").innerHTML = localStorage.getItem("webpage");
```

Comment:
https://www.loom.com/share/d4cd62e67604411a88b1f80e3d4d9a8b

work around fix

Reply:
Unnecessary. Please remove this fix. Task 85 was meant to be tested as extension in production.